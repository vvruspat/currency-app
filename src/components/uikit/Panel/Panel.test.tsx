import { render, screen } from "@testing-library/react";
import { Panel } from ".";

test("renders panel", () => {
  render(<Panel data-testid="test-panel" />, {});
  const PanelElement = screen.getByTestId("test-panel");
  expect(PanelElement).toBeInTheDocument();
  expect(PanelElement).toMatchSnapshot();
});
