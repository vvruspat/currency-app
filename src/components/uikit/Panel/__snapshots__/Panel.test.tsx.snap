// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders panel 1`] = `
<section
  class="panel"
  data-testid="test-panel"
>
  <div
    class="panel-content"
  />
</section>
`;
